.github/workflows/build.yml

```diff
@@ -32,45 +34,24 @@ jobs:
             export VERSION=${VERSION:1:${#VERSION}}
             echo "Building as ${VERSION}"
           fi
-
-          echo "##[set-output name=version;]$VERSION"
-        id: determine_version
-      - name: Build ${{ steps.determine_version.outputs.version }}
+          # store version in GitHub environment file
+          echo "version=$VERSION" >> $GITHUB_ENV
+      - name: Build ${{ env.version }}
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: :codyze-cli:build -x check --parallel -Pversion=$VERSION
-        id: build
-        env:
-          VERSION: ${{ steps.determine_version.outputs.version }}
+          arguments: :codyze-cli:build -x check --parallel -Pversion=${{ env.version }}
```

Comment:
@maximilian-galanis Do we still need to disable `check` in our Gradle execution?

Reply:
I remember now. Let's keep it as it is.